CHANGELOG.md

```diff
@@ -4,13 +4,13 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 ## Unreleased
 
 ### Added
-- 
+-
 
 ### Changed
-- 
+- **cf-layout** Updated main content spacing
```

Comment:
@Scotchester I actually wanted to ask about that. Do we know of any projects that this could break? It's a minor change, but I didn't expect the pseudo elem to be expecting 60px so it was a bit wonky till I found that.